front/SettingsForm.tsx

```diff
@@ -81,6 +82,7 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               rows={8}
               placeholder="Short bio about you"
+              data-cy="user-bio-settings"
```

Comment:
Consider using a more descriptive name for the data-cy attribute, such as 'user-bio-input'.